daemons/controld/controld_execd.c

```diff
@@ -1887,7 +1887,8 @@ do_lrm_invoke(long long action,
         lrmd_free_rsc_info(rsc);
 
     } else {
-        crm_err("Cannot perform operation %s of unknown type", crm_str(crm_op));
+        crm_err("Cannot perform operation %s of unknown type",
+                pcmk__s(crm_op, "(unspecified)"));
```

Comment:
if it doesn't fold into a sentence we might stay with the pointed brackets - kind of convention in my perception.

Reply:
the looking like xml thing hasn't been an issue for me so far but probably a valid argument (I wasn't aware of) ...